content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx

```diff
@@ -212,8 +212,22 @@ The generated Prisma Client has a reduced bundle size because the entire query e
 
 You can adjust the maximum number of database connections of each Data Proxy connection pool.
 
-To do so, in the Prisma Data Platform add the `connection_limit` parameter to the database connection string of the environment in your project.
+To do so, in the Prisma Data Platform, add the `connection_limit` parameter to the database connection string of the environment in your project.
+
+The _default_ value for the `connection_limit` in each Query Engine (QE) instance is _5_. This default value is determined using the calculation (`num_physical_cpus * 2 + 1`), where each QE instance consist of two CPUs.
 
 ```
-postgresql://user:password@host:5432/mydb?connection_limit=60
+prisma://region.prisma-data.com/?api_key=&connection_limit=10
 ```
+
+In the example connection string, the connection limit of _10_ is used as an arbitrary value. However, it's important to understand that this connection limit applies to each individual Query Engine (QE) instance.
+
+To ensure that your database's connections are not exhausted, it is recommended to set an appropriate value for the connection limit. This will help manage the surge in traffic effectively and prevent any connection-related issues.
+
+For example, if the `connection_limit` is set to _10_ and _5_ QE instances are
+created, a total of _50_ connections would be established with the database
+(`connection_limit * QE instances`). It is important to ensure that the database
+has sufficient capacity to handle this load of _50_ connections.
```

Comment:
it feels like switching the order of these two paragraphs makes more sense. the example directly relates to the sentence from the previous paragraph: "However, it's important to understand that this connection limit applies to each individual Query Engine (QE) instance."

```suggestion
For example, if the `connection_limit` is set to _10_ and _5_ QE instances are created, a total of _50_ connections would be established with the database (`connection_limit * QE instances`). It is important to ensure that the database has sufficient capacity to handle this load of _50_ connections.

To ensure that your database's connections are not exhausted, it is recommended to set an appropriate value for the connection limit. This will help manage the surge in traffic effectively and prevent any connection-related issues.
```

Reply:
(there also seem to have been arbitrary line breaks in one paragraph that I removed)

Reply:
Cleared out redundant line breaks, initially added them for readability. Those have been cleared in the latest commit

Reply:
Thanks Niko, I actually followed the conventions from the docs to stay consistent https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#recommended-connection-pool-size. The numbers over there are italicized :sweat_smile:, hence, maintained some consistency